lib/logstash/plugin.rb

```diff
@@ -111,9 +111,9 @@ def clear_watchdog
 
   public
   def inspect
-    if !@config.nil?
-      description = @config \
-        .select { |k,v| !v.nil? && (v.respond_to?(:empty?) && !v.empty?) } \
+    if !@params.nil?
+      description = @params \
```

Comment:
I would say clean them :-) they look strange to me, but at the end up to you :smile: